### ISSUE TITLE
Tooltip is not accessible on Window High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6896]
## Environment

> Operating system

Windows 10 latest

> Browser

Edge, Firefox

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description
- the tooltip icon is too low contrast
- the icon should receive a visible focus outline
- a border should be drawn around the dialog that opens

Firefox
![image](https://user-images.githubusercontent.com/40970507/93948116-2a9ec980-fd03-11ea-8684-f86837a8caac.png)

Edge
![image](https://user-images.githubusercontent.com/40970507/93948121-2ffc1400-fd03-11ea-9c62-dfe42dcf26e4.png)
